src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameService.java

```diff
@@ -214,6 +218,16 @@ private void runActiveTurn(Game currentGame, Integer cardId) throws InterruptedE
         // set sleep for all card flips.
 
         handleMatch(currentGame, currentTurn);
+
+        if (checkFinished(currentGame)){
+            finishGame(currentGame);
+            publishGamefinished(currentGame);
+            Thread.sleep(GameConstant.getFinishSleep());
+
+            resetGame(currentGame);
```

Comment:
I think we should probably create a separate endpoint for this and only reset the game if this is triggered by the frontend.

--> Can leave it like this for now imo and change if frontend wants/needs it